control-plane/slimctl/internal/cmd/controller/channel.go

```diff
@@ -21,6 +21,7 @@ func NewChannelCmd(opts *options.CommonOptions) *cobra.Command {
 		Short: "Manage SLIM channels",
 		Long:  `Manage SLIM channels`,
 	}
+	cmd.PersistentFlags().StringP(basicAuthKey, "b", "", "Basic auth key for the control plane authentication")
```

Comment:
Any reason not not add it to the root command?

Reply:
We can already configure the controller part of SLIM to accept basic credentials, so maybe it makes sense to have it at the root, so that it can be used out of the box also for node-connect 🤔 WDYT?